### DISCUSSION TITLE
Same clothes in multiple images

### DISCUSSION THREAD

**Discussion by satish-rathore** [lllyasviel/Fooocus#3741] [Q&A]
Hello, please help me find out how my model can wear the same dress in multiple photos.

**Comment by Yogesh-DevHub** [comment]
Upload the cloth image, use SAM to select the cloth, invert selection and generate image. 

for reference: https://youtube.com/shorts/TBrMCd9125U?si=0CikRuMnkEXCdnpX

**Reply by satish-rathore** [reply]
Thankyou sir for your reply, but in the video u send, model is in same pose,back ground change..
I need, different poses in same clothes..

**Reply by Yogesh-DevHub** [reply]
The video was just to get the idea what I wrote about. You have to upload the cloth/dress and generate the person inside the cloth, hands and legs visible around the dress. 

For different poses of character in same cloths, you will need to create a lora for that cloth and use it while generating image.

**Reply by satish-rathore** [reply]
Is there any detailed guide of anything available ?
